### PR TITLE
Add mknotebooks

### PR DESCRIPTION
**Pull request by apcamargo**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by apcamargo** [comment]
Hey @conda-forge/help-python 
This package is Linux/macOS-only. I used `skip: True  # [not win]` to skip Windows builds, but the checks are failing. Is there a way of skipping those tests?

**Comment by scopatz** [comment]
`skip: True # [not win]` means to skip mac and linux (because you are not skipping windows).  `skip: True # [win]` means to skip windows only

**Comment by apcamargo** [comment]
Thanks @scopatz ! This was a dump mistake. I think I copied this line from another recipe and overlooked it.

The recipe is building fine now.

File: recipes/mknotebooks/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
```
Comment (scopatz) [inline recipes/mknotebooks/meta.yaml]: since this is not noarch, the skip line should be `skip: true  # [win or py<36]`

Comment (apcamargo) [reply recipes/mknotebooks/meta.yaml]: Thanks

**Comment by apcamargo** [comment]
The osx build is now failing `TypeError: stat: path should be string, bytes, os.PathLike or integer, not NoneType`

**Comment by scopatz** [comment]
This seems liek a bug in conda-build.  CC @msarahan

**Comment by scopatz** [comment]
CC @conda-forge/core, can you confirm?

**Comment by ocefpaf** [comment]
> CC @conda-forge/core, can you confirm?

Yep. It is an old bug that is back b/c we removed the latest version of conda-build due to a new bug. Fun right?! See https://github.com/conda-forge/hypothesis-feedstock/pull/306#issuecomment-508064812

**Pull request closed without merging**